sloop_ros/Loc loc
string type
string label